README.md

```diff
@@ -33,6 +33,7 @@ You must specify a `deploymentMode` property in `config.yaml`.
 * `deploymentMode` (optional, default = `"SelectorSyncSet"`) - either "Direct" or "SelectorSyncSet".
 
 ## Direct Deployment
+> :warning: **Most cases don't require Direct Deployment** : use SelectorSyncSet instead
```

Comment:
Why? This warning does not tell me which cases canuse `direct`

Reply:
I am not sure what case does need Direct, but that for deploying a CR to the clusters I needed a selector-sync-set

Reply:
@cblecker maybe you can lend me a hand and help clarify this?

My issue was solved by not using direct and maybe the readme and be spruced up to make it easier to understand